docs/dev/src/rsh/step/index.md

```diff
@@ -299,7 +299,7 @@ where:
 + `{!rsh} PART_EXPR` is an expression that evaluates to a participant;
 + `{!rsh} PUBLISH_EXPR` is a syntactic arrow expression that is evaluated in a local step for the specified participant and must evaluate to an object that may contain a `msg` field, which may be of any type, and a `when` field, which must be a boolean;
 + (optional) `{!rsh} PAY_EXPR` is an expression that evaluates to a function parameterized over the `msg` value and returns a pay amount; if this component is left-out, it is synthesized to zero;
-+ (optional) `{!rsh} PAY_REQUIRE_EXPR` is an expression that evaluates to a function parameterized over the `msg` value and returns `void`. It may be used to make `require` claims about the `msg` value. If this component is left-out, it is synthesized to a no-op function.
++ (optional) `{!rsh} PAY_REQUIRE_EXPR` is an expression that evaluates to a function parameterized over the `msg` value and returns `null`. It may be used to make `{!rsh} require` claims about the `msg` value. If this component is left-out, it is synthesized to a no-op function.
```

Comment:
Maybe it would be better to say "evaluated for effect" like we do in `API_ASSUME_EXPR`